media/transcoder/src/main/java/com/example/transcoder/CreateJobWithStaticOverlay.java

```diff
@@ -0,0 +1,137 @@
+import com.google.cloud.video.transcoder.v1beta1.Overlay;
+import com.google.cloud.video.transcoder.v1beta1.Overlay.AnimationEnd;
+import com.google.cloud.video.transcoder.v1beta1.Overlay.AnimationStatic;
+import com.google.cloud.video.transcoder.v1beta1.Overlay.NormalizedCoordinate;
+import com.google.cloud.video.transcoder.v1beta1.TranscoderServiceClient;
+import com.google.cloud.video.transcoder.v1beta1.VideoStream;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+
+public class CreateJobWithStaticOverlay {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String inputUri = "gs://my-bucket/my-video-file";
+    String overlayImageUri = "gs://my-bucket/my-overlay-image.jpg"; // Must be a JPEG
+    String outputUri = "gs://my-bucket/my-output-folder/";
+
+    createJobWithStaticOverlay(projectId, location, inputUri, overlayImageUri, outputUri);
+  }
+
+  // Creates a job from an ad-hoc configuration and adds a static overlay to it.
+  public static void createJobWithStaticOverlay(
+      String projectId, String location, String inputUri, String overlayImageUri, String outputUri)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (TranscoderServiceClient transcoderServiceClient = TranscoderServiceClient.create()) {
+
+      VideoStream videoStream0 =
+          VideoStream.newBuilder()
+              .setCodec("h264")
+              .setBitrateBps(550000)
+              .setFrameRate(60)
+              .setHeightPixels(360)
+              .setWidthPixels(640)
+              .build();
+      AudioStream audioStream0 =
+          AudioStream.newBuilder().setCodec("aac").setBitrateBps(64000).build();
+      Overlay.Image overlayImage =
+          Overlay.Image.newBuilder()
+              .setUri(overlayImageUri)
+              .setResolution(NormalizedCoordinate.newBuilder().setX(1).setY(0.5).build())
+              .setAlpha(1)
+              .build();
+      Overlay.Animation animationStatic =
+          Overlay.Animation.newBuilder()
+              .setAnimationStatic(
+                  AnimationStatic.newBuilder()
+                      .setXy(NormalizedCoordinate.newBuilder().setX(0).setY(0).build())
+                      .setStartTimeOffset(Duration.newBuilder().setSeconds(0).build())
+                      .build())
+              .build();
+      Overlay.Animation animationEnd =
+          Overlay.Animation.newBuilder()
+              .setAnimationEnd(
+                  AnimationEnd.newBuilder()
+                      .setStartTimeOffset(Duration.newBuilder().setSeconds(10).build())
+                      .build())
+              .build();
+      Overlay overlay =
```

Comment:
nit: Add a newline before this to separate it from the setup steps before it.

Reply:
Done